src/base/bittorrent/torrentimpl.cpp

```diff
@@ -1443,6 +1443,22 @@ void TorrentImpl::setFirstLastPiecePriority(const bool enabled)
     m_session->handleTorrentNeedSaveResumeData(this);
 }
 
+void TorrentImpl::applyExcludedFileNamesFilters()
+{
+    QVector<BitTorrent::DownloadPriority> priorities = filePriorities();
```

Comment:
```suggestion
    const QVector<BitTorrent::DownloadPriority> priorities = filePriorities();
```
Please apply `const` to variable as much as possible, this apply to all files.

Reply:
I can't make this change since the priorities variable is changed further down, but I have added const in other places where it was missing.